src/views/product-tutorials-view/helpers/heading-helpers.ts

```diff
@@ -11,26 +11,23 @@ import { ProductViewBlock } from '../components/product-view-content'
  * TableOfContentsHeading objects, each representing
  * a section that will be rendered on the page.
  */
-function buildLayoutHeadings(
-  pageData: {
-    blocks: ProductViewBlock[]
-    showProductSitemap?: boolean
-  },
-  productName: string
-): TableOfContentsHeading[] {
+function buildLayoutHeadings(pageData: {
+  blocks: ProductViewBlock[]
+  showProductSitemap?: boolean
+}): TableOfContentsHeading[] {
   const { blocks, showProductSitemap } = pageData
 
   /**
    * Build an <h1> overview heading
    */
-  const overviewHeading = [getOverviewHeading(productName)]
+  const overviewHeading = [getOverviewHeading()]
```

Comment:
In mockups, the overview heading is simply "Overview", so there's no need to pass `productName`. This change cascades to a few different places in the code.